### ISSUE TITLE
Feature request: ability to clean up or turn off completely mount points shown location dialog

### ISSUE THREAD

**Issue by chpasha** [elfmz/far2l#1196]
Currently, far2l automatically detects and shows too many useless mount points in location dialog, for instance I've got /run and /boot/efi shown in it under Debian - a couple of absolutely useless locations for daily usage (but there are actually more of them that I don't care about). 
After discussing the problem in telegram chat, two suggestions come into mind (they are not mutually exclusive)
1) allow to disable automatic mount point detection completely via configuration option - I personally can totally live without them, since it is not difficult to add a couple of really useful paths to the bookmark list manually
2) mount.sh script already filters some of the system mount points, I think that this functionality could be extended with user supplied filters placed somewhere under  ~/.config/far2l , for instance call it locations.blacklist putting one filter per line there - imo a simple string comparison would be enough, no need to bother with regular expressions

what do you think?

**Comment by elfmz** [comment]
May be add button 'Hide item' in F4 dialog (which allows to edit hotkey) and add Button 'Show all hidden items' in F9 dialog?

**Comment by akruphi** [comment]
> May be add button 'Hide item' in F4 dialog (which allows to edit hotkey) and add Button 'Show all hidden items' in F9 dialog?

More intuitive useful is to add **Ctrl+H** for action _Hide hidden (by F4)_/_Show all_ similar to Viewer/Editor Shift+F8 dialog.

**Comment by chpasha** [comment]
yes, hiding items would also make sense - to tell the truth, before reading the help I thought that Del would do the same - remove item from list
